.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# What does this change
+_Give a summary of the change, and how it affects end-users. It's okay to copy/paste your commit messages._
+
+_For example if it introduces a new command or modifies a commands output, give an example of you running the command and showing real output here._
+
+# What issue does it fix
+Closes # _(issue)_
+
+_If there is not an existing issue, please make sure we have context on why this change is needed. See our Contributing Guide for [examples of when an existing issue isn't necessary][1]._
+
+[1]: https://getporter.org/src/CONTRIBUTING.md#when-to-open-a-pull-request
+
+# Notes for the reviewer
+_Put any questions or notes for the reviewer here._
+
+# Checklist
+- [ ] Did you write tests?
+- [ ] Did you write documentation?
+- [ ] Did you make any API changes? Update the corresponding API documentation.
+- [ ] If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️
```

Comment:
It's awkward to tell them to open a second PR against the porter repo, so maybe just leave that off for now? Otherwise we need to be clear it's a separate PR in the core Porter repo that should be opened after their PR here is merged.